src/core/components/ChatView.module.css

```diff
@@ -94,11 +94,13 @@
 .assistantContainer {
   margin-top: var(--spacing-sm);
   max-width: 80%;
+  width: 80%;
```

Comment:
This keeps the thinking container from starting small and growing as the text is streamed in it to 80%